src/All/README.md

```diff
@@ -3,6 +3,10 @@
 ```haskell
 All Boolean
 ```
+<details>
```

Comment:
We may want to do something like

```html
<details style="margin:30px">
  <summary style="margin-bottom:15px">Breakdown</summary>
  <p>The Monoid <code> All </code> outputs type <code> Boolean </code></p>
</details>
```
For the Markdown READMEs...Then we can add css to the Electric Docs so we can get better control on the responsiveness